public/content/documentation/native/controls/search.md

```diff
@@ -0,0 +1,292 @@
+   - A search field should behave similar to a typical input field
+   - Ensure the placeholder does not take the place of the programmatic name for the search field
+   - If auto results populate, an optional announcement can be added to the programmatic name so the user knows there are options to choose from below
+   - Search icon can be the persistent visible label instead of a text label
+
+### Name
+
+   - Programmatic name describes the purpose of the control.
+   - If visible text label exists, the programmatic name should match the visible text label or search icon meaning
+      - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognition.
+   - Placeholder text is not the programmatic name
+
+   - **UIKit**
+
+      - You can programmatically set the visible label with `setTitle()`.
+         - The search field title will overwrite the search field `accessibilityLabel`.
+      - If a visible label is not applicable in this case, set the search filed `accessibilityLabel` to the label of your choice.
+         - To do this in Interface Builder, set the label using the Identity Inspector
+      - To hide labels from VoiceOver programmatically, set the label's `isAccessibilityElement` property to `false`
+      - To hide labels from VoiceOver using Interface Builder, uncheck `Accessibility Enabled` in the Identity Inspector.
+
+   - **SwiftUI**
+
+      - If no visible label, use view modifier `accessibilityLabel(_:)`
+
+### Role
+
+   - When using non-native controls (custom controls), roles will need to be manually coded.
+
+   - **UIKit**
+
+      - Use `UISearchTextField`
+
+   - **SwiftUI**
+
+      - `searchable(text:placement:)`
+
+### Groupings
+
+   - Group visible label or icon with the input field, if applicable, to provide a programmatic name for the search field.
+
+   - **UIKit**
+
+      - Ensure that the child elements of the overarching view you want to group has their `isAccessibilityElement` properties set to `false`.
+      - Set `isAccessibilityElement` to `true` for the parent view. Then, adjust `accessibilityLabel` and `accessibilityTraits` accordingly.
+         - If the frame does not exist due to implmenting a custom text input field, use `accessibilityFrameInContainer` to set the custom control’s frame to the parent view’s container or view of your choice.
+            - You can also unionize two frames with `frame.union` (i.e. `titleLabel.frame.union(subtitleLabel.frame)`).
+      - Use `shouldGroupAccessibilityElement` for a precise order if the native order should be disrupted.
+      - Use `shouldGroupAccessibilityChildren` to indicate whether VoiceOver must group its children views. This allows making unique vocalizations or define a particular reading order for a part of the page.
+
+**SwiftUI**
```

Comment:
Looks like this one wasn't indented originally but should be. Or maybe I'm forgetting if we decided to just copy as-is? If so, ignore this comment.

Reply:
definitely meant to! :) will update!